packages/default/scss/skeleton/_layout.scss

```diff
@@ -22,13 +22,6 @@
     }
 
 
-    // Legacy alias
-    .k-placeholder-line {
-        @extend .k-skeleton !optional;
-        @extend .k-skeleton-text !optional;
-    }
```

Comment:
This is used in the Grid component virtualization. If you want to deprecate the legacy alias, you need to also update the other tests, where the Skeleton should be used, and provide product guidelines for the change in the themes issue.

Reply:
I've decided to revert the change, since it is affecting another component and will require more time to resolve.

Reply:
We still want to do this change and streamline the usage of the Skeleton, as this is the essence of the task itself.